changelog.d/5729.bugfix

```diff
@@ -0,0 +1 @@
+Fix worker log config not being set correctly.
```

Comment:
probably wants merging with the PR that broke it rather than being a new changelog entry?

Reply:
err yes, done